staging/src/k8s.io/apiserver/pkg/server/options/etcd.go

```diff
@@ -139,7 +139,7 @@ func (s *EtcdOptions) AddFlags(fs *pflag.FlagSet) {
 	}
 
 	fs.StringSliceVar(&s.EtcdServersOverrides, "etcd-servers-overrides", s.EtcdServersOverrides, ""+
-		"Per-resource etcd servers overrides, comma separated. The individual override "+
```

Comment:
I believe this is correct, although maybe not worded in the most helpful way.  `StringSliceVar` is documented as a comma separate list.  Note on the next line the clarifies that the individual overrides are semicolon separated.

Maybe the best thing here would to be provide a example that shows two etcd server overrides (comma separated) where one of the server overrides has multiple group/resource overrides (semicolon separated)?

Reply:
@mengqiy Do you mind adding an example like @jpbetz pointed out? thanks!

Reply:
I will add an example